deploy-dapp/deploy-to-azure.md

```diff
@@ -0,0 +1,58 @@
+
+1.  In a web browser, navigate to the Blockchain Workbench address, which is in the format _<https://{workbench> URL}.azurewebsites.net/_.
+2.  Sign in as a  [Blockchain Workbench administrator](https://docs.microsoft.com/en-gb/azure/blockchain/workbench/manage-users#manage-blockchain-workbench-administrators) to assign roles to users.
+3.  Select  _Applications > New_, which displays the  "New application"  pane.
+4.  Select  _Upload the contract configuration  >  Browse_  to locate the configuration file you created. The configuration file is automatically validated. Select the  _Show_  link to display any validation errors. Fix the validation errors before you deploy the application. Repeat the same for any smart contract code file by selecting _Upload the contract code >  Browse_  to locate the smart contract code file.
+5.  Click  _Deploy_  to create the blockchain application based on the configuration and smart contract files.
+
+Deploying the blockchain application takes a few minutes. When the deployment is finished, the new application is displayed in  _Applications_.
+
+To create a new contract, you need to be a member specified as a contract  **initiator**. For information defining application roles and initiators for the contract, read  [workflows in the configuration overview](https://docs.microsoft.com/en-gb/azure/blockchain/workbench/configuration#workflows). For information on assigning members to application roles, see  [add a member to application](https://docs.microsoft.com/en-gb/azure/blockchain/workbench/manage-users#add-member-to-application).
+
+1.  In the Blockchain Workbench application section, select the application tile that contains the contract you want to create. A list of active contracts is displayed.
+2.  To create a new contract, select  _New contract_.
+3.  The _New contract_ pane is displayed. Specify the initial parameters values, and click _Create_.
+
+## Modifying smart contracts
+
+Depending on the state the contract is in, members can take actions to transition to the next state of the contract. Azure defines actions as ["transitions"](https://docs.microsoft.com/en-gb/azure/blockchain/workbench/configuration#transitions) within a [state](https://docs.microsoft.com/en-gb/azure/blockchain/workbench/configuration#states). Members belonging to an allowed application or instance role for the transition can take the action.
+
+1.  In the _Blockchain Workbench_ application section, select the application tile that contains the contract to take the action.
+2.  Select the contract in the list, which displays details about the contract.
+3.  In the  _Action_  section, select _Take action_.
+4.  The details about the current state of the contract are displayed in a pane. Choose the action you want to take in the drop-down.
+5.  Select  _Take action_  to initiate the action.
+6.  If the action requires any parameters, specify the values for the action.
+7.  Select _Take action_ to execute the action.
```

Comment:
@srisankethu Feel like there needs to be some sort of conclusion here… Also, what does someone do with the rest of their dapp, such as the front end code etc? I know this may seem obvious, but let's provide an end-to-end article, even if it's a very brief explanation.

Reply:
Addressed the issue

Reply:
It's a pretty brief conclusion… And doesn't address my comment about what developers do with the rest of the dapp. But I'll address this myself.